eldritch/test_mythos.py

```diff
@@ -1901,6 +1904,64 @@ def testCannotStartRumorWhenOneExists(self):
     self.assertEqual(cultist.toughness(self.state, self.char), 1)
 
 
+class CityBonusTest(EventTest):
+  def setUp(self):
+    super().setUp()
+    self.mythos = Mythos6()
+    self.state.mythos.append(self.mythos)
+    self.advance_turn(0, "mythos")
+    self.resolve_until_done()
+    self.assertEqual(self.mythos, self.state.environment)
+
+  def testAppliesInCity(self):
+    self.assertTrue(isinstance(self.char.place, places.CityPlace))
+    bonus_check = Check(self.char, "sneak", -1)
+    self.state.event_stack.append(bonus_check)
+    self.resolve_until_done()
+    self.assertEqual(len(bonus_check.dice.roll), 4 - self.char.speed_sneak_slider)
+    penalty_check = Check(self.char, "will", 1)
+    self.state.event_stack.append(penalty_check)
+    self.resolve_until_done()
+    self.assertEqual(len(penalty_check.dice.roll), 4 - self.char.fight_will_slider)
+
+  def testDoesntApplyInOtherWorlds(self):
+    self.char.place = self.state.places["Pluto2"]
+    self.assertFalse(isinstance(self.char.place, places.CityPlace))
+    bonus_check = Check(self.char, "sneak", 0)
+    self.state.event_stack.append(bonus_check)
+    self.resolve_until_done()
+    self.assertEqual(len(bonus_check.dice.roll), 4 - self.char.speed_sneak_slider)
+    penalty_check = Check(self.char, "will", 0)
+    self.state.event_stack.append(penalty_check)
+    self.resolve_until_done()
+    self.assertEqual(len(penalty_check.dice.roll), 4 - self.char.fight_will_slider)
+
+
+class ReleaseMonstersTest(EventTest):
+  def testMonstersReleased(self):
```

Comment:
Add one test that validates that if the board is full, the user gets a choice of monsters to place in the outskirts?